integration/cli-tests/src/test/java/EnvSetup.sh

```diff
@@ -29,20 +29,20 @@ echo "Bittype : $bitType"
 extractCompressArchive() {
     echo "Extracting the files"
 
-    if [[ "${platform}" == "Linux" && "${bitType}" == "i586" ]]; then
+    if [[ ( "${platform}" == "Linux" ) && ( "${bitType}" == "i586" ) ]]; then
         tar -xvzf wso2mi-cli-$VERSION-linux-i586.tar.gz
 
-    elif [[ "${platform}" == "Linux" && "${bitType}" == "x86_64" ]]; then
+    elif [[ ( "${platform}" == "Linux" ) &&  ( "${bitType}" == "x86_64" ) ]]; then
         echo "Extracting the linux compressed archive "
         tar -xvzf wso2mi-cli-$VERSION-linux-x64.tar.gz
```

Comment:
Shall we set the platform version and bit type to some variable nad do tar -xvzf only once.